builder/ConfTargets.cmake

```diff
@@ -80,6 +80,9 @@ option( MFX_ENABLE_USER_DECODE "Enabled user decode plugins?" ON)
 option( MFX_ENABLE_USER_ENCODE "Enabled user encode plugins?" ON)
 option( MFX_ENABLE_USER_ENC "Enabled user ENC plugins?" ON)
 option( MFX_ENABLE_USER_VPP "Enabled user VPP plugins?" ON)
+option( MFX_ENABLE_VP9_VIDEO_DECODE "Enabled VP9 decoder?" ON)
```

Comment:
add empty line between MFX_ENABLE_USER_VPP and MFX_ENABLE_VP9_VIDEO_DECODE, please